core/templates/dev/head/pages/exploration_editor/ParameterMetadataService.js

```diff
@@ -28,7 +28,7 @@ oppia.factory('parameterMetadataService', [
     var PARAM_SOURCE_ANSWER = 'answer';
     var PARAM_SOURCE_CONTENT = 'content';
     var PARAM_SOURCE_FEEDBACK = 'feedback';
-    var PARAM_SOURCE_PARAM_CHANGES = 'param_changes';
+    var PARAM_SOURCE_PARAM_CHANGES = 'paramChanges';
```

Comment:
Yeah my bad, I changed it while experimenting and forgot to change it back--pretty sure it shouldn't be changed